mungegithub/submit-queue/deployment/kubernetes/configmap.yaml

```diff
@@ -156,6 +156,9 @@ data:
     ci-kubernetes-e2e-gce-1.4-1.5-upgrade-cluster,\
     ci-kubernetes-e2e-gce-1.4-1.5-upgrade-cluster-new,\
     ci-kubernetes-e2e-gce-1.4-1.5-upgrade-master"
+  # SQ-Blocking Jobs
+  # Please also update sq-blocking dashboard in testgrid config for test-infra presubmit to pass (@krzyzacy)
+  # https://github.com/kubernetes/test-infra/blob/master/testgrid/config/config.yaml
```

Comment:
Implicit dependencies between files across repos? Yummy :-)

Also, I don't have much knowledge of test-grid config, and I think I would be confused what to do if I had to change it.

> Please also update sq-blocking dashboard in testgrid config

Searching "sq-blocking" in testgrid config ... no match found, wat?